src/decode.rs

```diff
@@ -4387,74 +4387,54 @@ pub(crate) unsafe fn rav1d_decode_frame_init(
     let mut uv_stride = f.cur.stride[1];
     let has_resize = (frame_hdr.size.width[0] != frame_hdr.size.width[1]) as c_int;
     let need_cdef_lpf_copy = (c.tc.len() > 1 && has_resize != 0) as c_int;
-    if y_stride * f.sbh as isize * 4 != f.lf.cdef_buf_plane_sz[0] as isize
-        || uv_stride * f.sbh as isize * 8 != f.lf.cdef_buf_plane_sz[1] as isize
-        || need_cdef_lpf_copy != f.lf.need_cdef_lpf_copy
-        || f.sbh != f.lf.cdef_buf_sbh
-    {
-        rav1d_free_aligned(f.lf.cdef_line_buf as *mut c_void);
-        let mut alloc_sz: usize = 64;
-        alloc_sz += (y_stride.unsigned_abs() * 4 * f.sbh as usize) << need_cdef_lpf_copy;
-        alloc_sz += (uv_stride.unsigned_abs() * 8 * f.sbh as usize) << need_cdef_lpf_copy;
-        f.lf.cdef_line_buf = rav1d_alloc_aligned(alloc_sz, 32) as *mut u8;
-        let mut ptr = f.lf.cdef_line_buf;
-        if ptr.is_null() {
-            f.lf.cdef_buf_plane_sz[1] = 0;
-            f.lf.cdef_buf_plane_sz[0] = f.lf.cdef_buf_plane_sz[1];
-            return Err(ENOMEM);
-        }
+    let mut alloc_sz: usize = 64;
+    alloc_sz += (y_stride.unsigned_abs() * 4 * f.sbh as usize) << need_cdef_lpf_copy;
+    alloc_sz += (uv_stride.unsigned_abs() * 8 * f.sbh as usize) << need_cdef_lpf_copy;
+    // TODO: Fallbile allocation. We need to do the following on allocation
+    // failure:
+    // f.lf.cdef_buf_plane_sz = [0, 0];
```

Comment:
```suggestion
    // TODO: Fallible allocation
```

We can take out that comment since that field is gone now.  Also, typo in fallible.